kqueen/auth/common.py

```diff
@@ -1,15 +1,28 @@
 """Authentication methods for API."""
 
 from kqueen.config import current_config
-from kqueen.models import Organization, User
+from kqueen.models import Organization
+from kqueen.models import User
 from uuid import uuid4
 
 import bcrypt
+import importlib
 import logging
 
 logger = logging.getLogger('kqueen_api')
 
 
+def get_auth_instance(name):
+    config = current_config()
+
+    auth_config = config.get("AUTH", {}).get(name, {})
+
+    module = importlib.import_module('kqueen.auth')
+    auth_class = getattr(module, name)
+
+    return auth_class(**auth_config)
```

Comment:
Considering `auth_class` can be None, you could handle the possible `NoneType not callable`  error here